server/libs/userlibs.js

```diff
@@ -42,15 +42,32 @@ const register = function(userData) {
     })
 };
 
-const logIn = function(body) {
-    return new Promise((resolve, reject) => {
-        User.findOne({email : body.email, deleted : false}).then((user) => {
-            if (!user) {
+const getUser = function (query) {
+    return new Promise((resolve,reject) => {
+        User.findOne(query).then((user) => {
+            if (user) {
+                resolve(user);
```

Comment:
no message no status no data??

Reply:
This is a generic function. This is not a function for routing. So that's why only sending back the token which is created.